docs/design/integration_nova.adoc

```diff
@@ -20,16 +20,194 @@ We would want to integrate with Nova to support the same set of user operations,
 
 == Review of OpenStack Workflow
 
+nova call neutron apis
+|===
+|Name |method |params |fields |neutron api |alcor status
+
+|show_network
+|get
+|net_id
+|provider:physical_network,provider:network_type
+|/v2.0/networks/{net_id}
+|/v4/{project_id}/vpcs/{vpcid}
+
+|show_network
+|get
+|net_id
+|dns_domain
+|/v2.0/networks/{net_id}
```

Comment:
Neutron 2.0 APIs doesn't need to provide a project id as it generates the project id based on the Authentication token in the X-Auth-Token request header, when Keystone is enabled. 

https://docs.openstack.org/api-ref/network/v2/
"The Networking API v2.0 uses the OpenStack Identity service as the default authentication service. When Keystone is enabled, users that submit requests to the OpenStack Networking service must provide an authentication token in X-Auth-Token request header. You obtain the token by authenticating to the Keystone endpoint.

When Keystone is enabled, the project_id attribute is not required in create requests because the project ID is derived from the authentication token."

@Gzure We will make changes to API Gateway to support Authentication & Authorization with KeyStone, and support same way how project id is generated based on X-Auth-Token in API Gateway. Can you include details in this design doc as well?

Reply:
@xieus neutron also verify nova call api by keystone, Do we need add this to alcor?

Reply:
@Gzure yes we do. Let me create a new request to track this new feature request. https://github.com/futurewei-cloud/alcor/issues/207